spike-storage-queue/server-issue-sas/index.js

```diff
@@ -0,0 +1,44 @@
+'use strict'
+
+require('dotenv').config()
+const azure = require('azure-storage')
+
+const generateSasToken = () => {
+  const storageConnection = process.env.AZURE_STORAGE_CONNECTION_STRING
+  if (!storageConnection) {
+    process.exitCode = -1
+    console.error('env var $AZURE_STORAGE_CONNECTION_STRING is required')
+  }
+
+  const queueName = process.env.QUEUE_NAME || 'sastest'
+
+  var queueService = azure.createQueueService(storageConnection)
+
+  // Create a SAS token that expires in an hour
+  // Set start time to five minutes ago to avoid clock skew.
+  var startDate = new Date()
+  startDate.setMinutes(startDate.getMinutes() - 5)
+  var expiryDate = new Date(startDate)
+  expiryDate.setMinutes(startDate.getMinutes() + 60)
```

Comment:
For the actual work, we would probably want the token to live until the end of the check window - the same as the JWT token.

Reply:
I would assume we would want to rely on moment for dates to be consistent for the real work.

Reply:
absolutely on both points.  this is a very rough POC, just to get something working.

You guys can add all the finesse and polish 🥇